src/main/java/edu/harvard/iq/dataverse/util/json/JsonPrinter.java

```diff
@@ -874,4 +875,33 @@ public static JsonObjectBuilder mapToObject(Map<String,String> in) {
         in.keySet().forEach( k->b.add(k, in.get(k)) );
         return b;
     }
+
+
+    /**
+     * Get signposting from SignpostingResources
+     *
+     * @param sr corresponding SignpostingResources
+     * @return json linkset
+     */
+    public static JsonArrayBuilder jsonLinkset(SignpostingResources sr) {
```

Comment:
Why is this needed? If you have an sr, can't you call sr.getJsonLinkset() yourself?